tasks/models.py

```diff
@@ -139,6 +138,16 @@ class Task(models.Model):
         blank=True,
     )
 
+    repeats = models.PositiveBigIntegerField(
+        _("Hoe vaak wordt deze taak herhaald?"),
```

Comment:
Suggestion:
"Hoe vaak moet de participant deze taak doen?"

But then, again, with proper Dutch/English

In addition, again, a help-text with more details could be worth looking at.